static/src/components/builder/components/ComponentRender/ComponentRender.vue

```diff
@@ -85,6 +87,10 @@ export default {
           attrs: { id: 'test' },
           props: { checkboxList: this.form.checkboxList }
         };
+      } else if (this.selectedComponent === 'conditional-display-creator') {
+        return {
+          name: 'conditionaldisplay-creator'
```

Comment:
Why did you hyphenate `conditional display creator` two different ways? It's confusing.

Reply:
I noticed this is not used will remove it.